02.calendar/cal.js

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env node
+
+import minimist from "minimist";
+import _ from "lodash-es";
+
+const runCalendar = () => {
+  const date = setDate();
+  if (!date.year || !date.month) {
+    console.error(
+      "不正な入力です。年は半角自然数・月は半角1~12で入力してください。",
+    );
+    return;
+  }
+  const weeks = buildWeeks(date.year, date.month);
+  const calendar = formatCalendar(date.year, date.month, weeks);
+  console.log(calendar);
+};
+
+const setDate = () => {
+  const { y, m } = minimist(process.argv.slice(2));
+  const inputYear = y,
+    inputMonth = m;
+  const today = new Date();
+  const year = setYear(inputYear, today);
+  const month = setMonth(inputMonth, today);
+  const baseData = { year: year, month: month };
```

Comment:
プロパティの名前と値が同じときは省略記法を使ってください。ところで、data という名前は特に何も表していないので、具体的な名前を付けたほうが良いと思います。base についても、この関数の文脈においては何に対する base なのかよく分からないです。

```suggestion
  const baseData = { year, month };
```

Reply:
省略記法を理解しました。
`base`も含め、変数の命名についての疑問点を提出ページの質問に記入いたしました。